lib/ui/common/Alert.dart

```diff
@@ -0,0 +1,226 @@
+      actionTextStyle: actionTextStyle ?? this.actionTextStyle,
+    );
+  }
+}
+
+class _DefaultStyle extends AlertStyle {
+  final backgroundColor = const Color(0xffffffff);
+  final titleTextStyle = const TextStyle(
+    color: Color(0xff1a1b46),
+    fontSize: 27,
+    fontWeight: FontWeight.bold,
+  );
+  final detailTextStyle = const TextStyle(
+    color: Color(0xff1a1b46),
+    fontSize: 17,
+    height: 1.1,
+    fontWeight: FontWeight.normal,
+  );
+  final actionBackgroundColor = const Color(0xff24d900);
+  final destructiveActionBackgroundColor = const Color(0xffff6060);
+  final actionTextStyle = const TextStyle(
+    color: Color(0xffffffff),
+    fontSize: 15,
+  );
+
+  const _DefaultStyle({
+    Color backgroundColor,
+    TextStyle titleTextStyle,
+    TextStyle detailTextStyle,
+    Color actionBackgroundColor,
+    Color destructiveActionBackgroundColor,
+    TextStyle actionTextStyle,
+  });
+}
+
+const AlertStyle _defaultStyle = const _DefaultStyle();
+
+class Alert extends StatelessWidget {
+  final AlertViewModel _viewModel;
+  final AlertStyle _style;
+
+  static present (Alert alert, BuildContext context) {
+    return showDialog<bool>(
+        context: context,
+        barrierDismissible: false,
+        builder: (_) => alert,
+    );
+  }
+
+  const Alert(
+      {Key key,
+      AlertViewModel viewModel,
+      AlertStyle style = _defaultStyle})
+      : this._viewModel = viewModel,
+        this._style = style,
+        super(key: key);
+
+  @override
+  Widget build(BuildContext context) {
+    return Material(
+      type: MaterialType.transparency,
+      child: Center(
+        child: Container(
+          margin: _Constants.alertEdgePadding,
+          decoration: BoxDecoration(
+            borderRadius: _Constants.cornerRadius,
+            color: _style.backgroundColor,
+          ),
+          child: Padding(
+            padding: _Constants.alertInnerPadding,
+            child: Column(
+              crossAxisAlignment: CrossAxisAlignment.start,
+              mainAxisSize: MainAxisSize.min,
+              children: <Widget>[
+                Text(
+                  _viewModel.titleText,
+                  style: _style.titleTextStyle,
+                ),
+                SizedBox(
+                  height: _Constants.titleLineSpace,
+                ),
+                Column(
+                  mainAxisSize: MainAxisSize.min,
+                  children: <Widget>[
+                    Text(
+                      _viewModel.detailText,
+                      style: _style.detailTextStyle,
+                    ),
+                    SizedBox(
+                      height: _Constants.detailLineSpace,
+                    ),
+                    Row(
+                      children: _buildAction(context),
+                    ),
+                  ],
+                ),
+              ],
+            ),
+          ),
+        ),
+      ),
+    );
+  }
+
+  confirmDialog(BuildContext context) {
```

Comment:
Not used?

Reply:
Yeah, i moved that to the Alert.present, but i forget to remove from here, Thanks !